### PR TITLE
Page objects2

### PR DESCRIPTION
**Pull request by Remi-Mignon**

### CODE REVIEW

File: pageObject/locators.py [hunk 1]
```diff
@@ -3,6 +3,19 @@ class Locators:
     amazonURL = "https://amazon.fr"
     cookie = "#sp-cc-accept"
 
+    #HomePage
```
Comment (barranquerox) [inline pageObject/locators.py:7]: It's clearer if you put the selectors in the classes where they are used.

File: test_amazon.py [hunk 2]
```diff
@@ -11,9 +16,27 @@ def test_amazon():
...
+    confirmation = ConfirmationPage(driver)
+    confirmation.openCart()
+
+    quantity = "2"
+    cart = CartPage(driver)
+    cart.setQuantity(quantity)
+
+    assert cart.getQuantity() == quantity
+
+    base.setdown()
```
Comment (barranquerox) [inline test_amazon.py]: rename the `setdown` function to `teardown`

File: pageObject/productPage.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from .locators import Locators
+from .basePage import BasePage
+
+
+class ProductPage(BasePage):
+
+    def addToCart(self):
+        self.clic(Locators.addToCart)
```
Comment (barranquerox) [inline pageObject/productPage.py]: rename to `click`

File: pageObject/locators.py [hunk 4]
```diff
@@ -3,6 +3,19 @@ class Locators:
...
     menu = "a#nav-hamburger-menu"
     booksMenu = ".hmenu-item[data-menu-id='10']"
-    allBooksMenu = "#hmenu-content > ul.hmenu.hmenu-visible.hmenu-translateX > li:nth-child(3) > a"
\ No newline at end of file
+    allBooksMenu = "#hmenu-content > ul.hmenu.hmenu-visible.hmenu-translateX > li:nth-child(3) > a"
+
+    #BooksPage
+    firstNewBook = "#a-page > div.a-container > div.a-row.apb-browse-two-col-center-pad > div.a-column.a-span12.aok-float-right.apb-browse-col-pad-left.apb-browse-two-col-center-margin-right > div:nth-child(4) > div > div.a-section.octopus-pc-card.octopus-best-seller-card > div.a-section.octopus-pc-card-content > ul > li:nth-child(2)"
```
Comment (barranquerox) [inline pageObject/locators.py]: this selector is too complicated, too long, too fragile. Please find one which is simpler.

**Review by barranquerox — CHANGES_REQUESTED** [review]

File: pageObject/locators.py (hunk 1 above)
Comment (Remi-Mignon) [reply pageObject/locators.py:7]: But if the selector changes for some reasons, you need to find in which class(es) it is used

**Pull request merged**